maven-xml-impl/pom.xml

```diff
@@ -39,12 +39,6 @@ under the License.
       <groupId>org.eclipse.sisu</groupId>
       <artifactId>org.eclipse.sisu.plexus</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.codehaus.plexus</groupId>
-      <artifactId>plexus-utils</artifactId>
```

Comment:
That looks just wrong.  The change works only because sisu-plexus has a dependency on plexus-utils, but the plexus-utils dependency is a requiered dependency and should not be only transitive.
This may be the same for other removal, but if there's any direct import, the dependency should be kept as a direct dependency.

Reply:
Basically, it should not add any new warning when running `mvn dependency:analyze` ...

Reply:
Ok, apologies; I've reverted ba610510b4b4e6c19aed5979908cf8daf46aaa7e in 687912dc62496d655f048c3402be2615ad6bddef, which restores the dropped dependencies.